Changes

```diff
@@ -1,5 +1,31 @@
 Release history for Zonemaster component Zonemaster-Backend
 
+v6.1.0 2020-11-06 (public release version)
+
+ [Features]
+ - Adds full support of a light weight SQLite database backend
+   as an alternative to MariaDB or PostgreSQL database daemons
+   (#695, #690, #177
```

Comment:
Missing closing bracket.

Reply:
Corrected all cases of missing parentheses.